edge-functions/feature-flag-optimizely/README.md

```diff
@@ -1,49 +1,63 @@
+This example shows how to use optimizely to conduct experiments in edge-functions.
 
-### Identity Management
-Out of the box, Optimizely's Full Stack SDKs require a user-provided identifier to be passed in at runtime to drive experiment and feature flag decisions. This example generates a unique id, stores it in a cookie and reuses it to make the decisions sticky. Another common approach would be to use an existing unique identifier available within your application.
+## Demo
 
-### Bucketing
-For more information on how Optimizely Full Stack SDKs bucket visitors, see [here](https://docs.developers.optimizely.com/full-stack/v4.0/docs/how-bucketing-works).
+https://feature-flag-optimizely.vercel.app
 
 ## One-Click Deploy
 
-**Note:** Before clicking `Deploy`, Please follow the section [Set up Optimizely](#set-up-optimizely) to create an account, set up feature flags and obtain the Optimizely *SDK Key*.
+**Note:** Before clicking `Deploy`, Please follow the section [Set up Optimizely](#set-up-optimizely) to create an account, set up feature flags and obtain the Optimizely _SDK Key_.
 
 [![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/clone?repository-url=https://github.com/optimizely/vercel-examples/tree/main/edge-functions/feature-flag-optimizely&env=OPTIMIZELY_SDK_KEY&project-name=feature-flag-optimizely&repository-name=feature-flag-optimizely)
 
 ## Run Locally
 
-**Note:** Before running locally, Please follow the section [Set up Optimizely](#set-up-optimizely) to create an account, set up feature flags and obtain the Optimizely *SDK Key*.
+**Note:** Before running locally, Please follow the section [Set up Optimizely](#set-up-optimizely) to create an account, set up feature flags and obtain the Optimizely _SDK Key_.
 
 1. generate the Next JS app using this example
-    ```
-    npx create-next-app --example https://github.com/optimizely/vercel-examples/tree/main/edge-functions/feature-flag-optimizely feature-flag-optimizely
-    ```
-    or    
-    ```
-    yarn create-next-app --example https://github.com/optimizely/vercel-examples/tree/main/edge-functions/feature-flag-optimizely feature-flag-optimizely
-    ```
-
-2. create a local environment file from example and add your optimizely *SDK Key*.
-    ```
-    cp .env.example .env.local
-    ```
+
+   ```
+   npx create-next-app --example https://github.com/optimizely/vercel-examples/tree/main/edge-functions/feature-flag-optimizely feature-flag-optimizely
+   ```
+
+   or
+
+   ```
+   yarn create-next-app --example https://github.com/optimizely/vercel-examples/tree/main/edge-functions/feature-flag-optimizely feature-flag-optimizely
+   ```
+
+2. create a local environment file from example and add your optimizely _SDK Key_.
+
+   ```
+   cp .env.example .env.local
+   ```
 
 3. Run locally
-    ```
-    npm run dev
-    ```
-    or
-    ```
-    yarn dev
-    ```
+   ```
+   npm run dev
+   ```
+   or
+   ```
+   yarn dev
+   ```
```

Comment:
@gbibeaul Let's make this be a copy paste of what we have in all other examples, like using a single code block, use the same paragraphs, etc. It should look like our template + optimizely.